build.sbt

```diff
@@ -229,6 +229,7 @@ def runTaskOnlyOnSparkMaster[T](
 }
 
 lazy val connectCommon = (project in file("spark-connect/common"))
+  .disablePlugins(JavaFormatterPlugin)
```

Comment:
so, this is opt-out? We cannot have delta-kernel-api and delta-kernel-defaults opt-in?

Reply:
I don't know sbt well enough, so from first experience it seems that projects need to opt-out from using a certain plugin

Reply:
Yeah; i couldn't figure this out either